evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -266,6 +266,15 @@ contract Operator is
     return address(s_LinkToken);
   }
 
+  function foreward(address _to, bytes calldata _data)
+    public
+    onlyAuthorizedNode()
```

Comment:
`onlyAuthorizedNode` does two storage reads:

```
  modifier onlyAuthorizedNode() {
    require(s_authorizedNodes[msg.sender] || msg.sender == owner, "Not an authorized node to fulfill requests");
    _;
  }
```

We could save 800 gas on each call by getting rid of the `owner` part.

Reply:
@kaleofduty Doesn't it short-circuit and avoid the second storage read? As the owner it seems helpful to allow the owner use the method as well. Alternatively, we could break out another forwarding method for the owner and avoid that first gas read.

Reply:
It does, not sure what was going on with my brain yesterday eve 🤦 
So never mind.

Reply:
oh I like that. And the owner just has to add themselves as an authorizedNode if they want to use the function?

Reply:
@kaleofduty - is the || lazy evaluated in solidity? Aka if the first part is true will solidity avoid the 2nd storage read?